### DISCUSSION TITLE
When using Cpanel to host your project, how do you set up SSL inside the laravel .env file to connect to Planet Scale?

### DISCUSSION THREAD

**Discussion by LeratoLetwaba** [planetscale/discussion#504] [Q&A]
When working locally, I was able to connect by getting the Mozilla certificate https://curl.se/docs/caextract.html. Now I'm attempting to host the project with CPanel and im failing

**Comment by orware** [comment]
Hello @LeratoLetwaba,

If I'm remembering correctly, cPanel is typically hosted on a CentOS/Red Hat distribution so using the following path for Laravel's `MYSQL_ATTR_SSL_CA` value within the `.env` file will likely be what's needed:
https://planetscale.com/docs/concepts/secure-connections#redhat--fedora--centos--mageia--vercel--netlify 

You could also choose to use something like the Mozilla certificate file, but you would have to also make an adjustment to Laravel's `config/database.php` config file since the path it uses for referencing `MYSQL_ATTR_SSL_CA` expects an absolute path to be provided.
